### PR TITLE
Fix up release-0.6 branch 

### PR DESCRIPTION
**Pull request by lukas-bednar**
In my opinion this is what should land in release-0.6 branch to make our tests work for KubeVirt v0.6.3.

I DO NOT demand to get this PR merged, it is just demonstration what is missing there.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-  version: 11459a886d9cd66b319dac7ef1e917ee221372c9
+  version: v1.4.0
 - package: github.com/onsi/gomega
   version: dcabb60a477c2b6f456df65037cb6708210fbb02
 - package: kubevirt.io/kubevirt
-  version: v0.7.0
+  version: v0.6.3
   repo:    https://github.com/kubevirt/kubevirt.git
+- package: kubevirt.io/qe-tools
+  version: v0.1.1
```
Comment (nellyc) [inline glide.yaml]: any reason not to take 0.1.2?

File: glide.yaml [hunk 2]
```diff
@@ -1,9 +1,12 @@
 package: kubevirt.io/kubevirt-ansible
 import:
 - package: github.com/onsi/ginkgo
-  version: 11459a886d9cd66b319dac7ef1e917ee221372c9
+  version: v1.4.0
```
Comment (nellyc) [inline glide.yaml]: any reason not to take v1.6.0?
relates to the below qe-tools version i guess

**Comment by nellyc** [comment]
lgtm

**Comment by lukas-bednar** [comment]
Actually we could remove both (qe-tools & ginkgo) from glide.yaml, since these are already dependencies of kubevirt ...

**Comment by lukas-bednar** [comment]
@gbenhaim  I am good with the change at this moment.

File: hack/common.sh [hunk 3]
```diff
@@ -7,6 +7,8 @@ KUBEVIRT_ANSIBLE_DIR="$(
 OUT_DIR=$KUBEVIRT_ANSIBLE_DIR/_out
 TESTS_OUT_DIR=$OUT_DIR/tests
 
+FUNC_TEST_ARGS="${FUNC_TEST_ARGS:--test.timeout 60m --junit-output=junit.xml}"
```
Comment (lukas-bednar) [inline hack/common.sh]: Need to change name of report

**Comment by lukas-bednar** [comment]
It seems that we will have to cherry-pick dropping ocp-3.9 for this branch as well ...

**Comment by lukas-bednar** [comment]
Cherry-picked here #361

**Comment by lukas-bednar** [comment]
@gbenhaim I can make it part of this PR

**Comment by lukas-bednar** [comment]
@gbenhaim it looks good, results were collected and processed by jenkins

**Comment by gbenhaim** [comment]
@lukas-bednar Nice
Feel free to merge.

**Pull request merged**